frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.model.ts

```diff
@@ -43,6 +43,21 @@ export interface UpdateFeatureFlagStatusRequest {
   status: FEATURE_FLAG_STATUS;
 }
 
+export enum UpsertFeatureFlagListAction {
+  ADD = 'add',
+  EDIT = 'edit',
+}
+
+export interface UpsertFeatureFlagListParams {
+  sourceList: any; // TODO define me
+  action: UpsertFeatureFlagListAction;
+}
+
+export enum LIST_OPTION_TYPE {
+  INDIVIDUAL = 'Individual',
+  SEGMENT = 'Segment',
+}
```

Comment:
Do we use any naming conventions for the `enum` types? `ALL_CAPS` or `UpperCamelCase`? Also, do we need to define these in a separate file (`types/src/Experiment/enums.ts`)? I have the same question for the `interface` type.

Reply:
our various type enums and constants such are definitely not standardized / organized very well. I tend to use all caps for any kind of enum or constant that produces a value, but upper-camel-case for type interfaces

Reply:
you have used `UpsertFeatureFlagListAction` enum can you change that...